src/dotnet/CommonLocalizableStrings.cs

```diff
@@ -179,5 +179,9 @@ internal class CommonLocalizableStrings
         public const string SpecifiedNameExists = "Specified name {0} already exists. Please specify a different name.";
         public const string SpecifiedAliasExists = "Specified alias {0} already exists. Please specify a different alias.";
         public const string MandatoryParameterMissing = "Mandatory parameter {0} missing for template {1}. ";
+
+        public const string ProjectNotCompatibleWithFramework = "Project `{0}` is not compatible with `{1}`.";
```

Comment:
@blackdwarf I'm sure you'll agree. We need to think about the end-to-end UX here. "I'm broken, great. now what?"